nextflow.config

```
@@ -32,6 +32,7 @@ params {
 
     // Blobtoolkit server webaddress
     btk_location                = null
+    btk_online_location         = "https://blobtoolkit.genomehubs.org/api/v1/dataset/id/CALUEP01"
```

nextflow.config shouldn't have any specific value set
```suggestion
    btk_online_location         = null
```

---

That's fair, I must have left it in there from testing. Bad practise in it's own right i know.